src/test/resources/com/puppycrawl/tools/checkstyle/grammars/javadoc/javadocTags/expectedAsteriskInJavadocInlineTagAst.txt

```diff
@@ -2,10 +2,10 @@ JAVADOC -> JAVADOC [0:0]
 |--JAVADOC_INLINE_TAG -> JAVADOC_INLINE_TAG [0:0]
 |   |--JAVADOC_INLINE_TAG_START -> { [0:0]
 |   |--LINK_LITERAL -> @link [0:1]
-|   |--NEWLINE -> \r\n [0:6]
```

Comment:
It has happened since I've copied the expected results from idea after executing the tests. It doesn't make any difference anyways since before comparing the trees we [replace](https://github.com/checkstyle/checkstyle/blob/master/src/test/java/com/puppycrawl/tools/checkstyle/BaseCheckTestSupport.java#L195) the `CLRF` with `LF`